scripts/roles.coffee

```diff
@@ -83,37 +89,43 @@ parseCommaSeparatedString = (string) ->
 removeObjects = (source, itemsToRemove) ->
   _.reject(source, (item) ->
     itemsToRemove.indexOf(item) > -1)
+    
+typeIsArray = Array.isArray || ( value ) -> return {}.toString.call( value ) is '[object Array]'
```

Comment:
How come we need the right side of this `||`? Can you pull this out into it's own named method so it self documents?